src/azure-cli/HISTORY.rst

```diff
@@ -3,6 +3,11 @@
 Release History
 ===============
 
+**IoT Central**
+
+* az iotcentral: Deprecated
+* Added 'az iot central' command module
+
```

Comment:
I added history notes in PR description, you can remove them from this file. History is now generated with PR title or PR description, more details can be found [here](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

Reply:
Thanks